specification/cognitiveservices/AnomalyDetector/multivariate/routes.tsp

```diff
@@ -31,7 +31,7 @@ op MultivariateServiceAction<TParams, TResponse>(
 For asynchronous inference, get a multivariate anomaly detection result based on the
 resultId value that the BatchDetectAnomaly API returns.
 """)
-op GetMultivariateBatchDetectionResult(
+op getMultivariateBatchDetectionResult(
```

Comment:
Why did you change the case of these operations in the TypeSpec?  New linting rule?

Why **didn't** the case of the operations in the generated swagger change?  I still see `Multivariate_GetMultivariateBatchDetectionResult` in the swagger.

Reply:
This is a new linting rule.  The swagger-emitter automatically changes the operation names to PascalCase.  The operation name has no impact on what appears on the wire, only potentially on client emitters, which are expecting camelCase as input for operations, and will normally transform this into what is idiomatic for their language. (camelCase in Python, TypeScript, Pascal case in C#, Java, etc.)